tests/test_characters.py

```diff
@@ -33,6 +35,16 @@ def test_possible_encoding():
         assert possible_encoding(char, 'latin-1')
 
 
+def test_byte_order_mark():
+    eq_(fix_encoding('ï»¿'), '\ufeff')
+
+
+def test_emoji_variation_selector():
+    # The hearts here are explicitly marked as emoji using the variation
+    # selector U+FE0F. This is not weird.
+    eq_(sequence_weirdness('❤️❤️❤️❤️❤️❤️❤️❤️❤️'), 0)
```

Comment:
Would the test still work if the string were spelled, e.g., `'❤ \ufe0f' * 9`?  Might be plainer to the casual observer (even without the comment).